mysql-test/suite/innodb_fts/r/plugin_debug.result

```diff
@@ -0,0 +1,306 @@
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+mysql -database' IN BOOLEAN MODE);
+id	title	body
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('>mysql <database ~search' IN BOOLEAN MODE);
+id	title	body
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+2	MySQL Tutorial	DBMS stands for MySQL good one DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+(mysql database) -engine' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+2	MySQL Tutorial	DBMS stands for MySQL good one DataBase ...
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+(mysql database -engine' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"mysql"' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+2	MySQL Tutorial	DBMS stands for MySQL good one DataBase ...
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"for mysql"' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+2	MySQL Tutorial	DBMS stands for MySQL good one DataBase ...
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"mysql database"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"full text search"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"full text search engine"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+"dbms stands" -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands") -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands" search) -good' IN BOOLEAN MODE);
+id	title	body
+5	1001 MySQL Tricks	How to use full-text search engine
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands" "full text") -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
```

Comment:
In MySQL 5.7, this and many other searches for `full text` or `full text search` would also return another row:
```
5	1001 MySQL Tricks	How to use full-text search engine
```
This difference is not explained in the commit message. Is it a difference in tokenization? Is there a way to make `-` considered a word separator? Could additional searches for `full-text` be added to demonstrate that the records would be found?